### PR TITLE
fix for https://github.com/wso2/docs-apim/issues/1129

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
fix for https://github.com/wso2/docs-apim/issues/1129

### CODE REVIEW

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# Configuring Keystores in API-M Analytics
+
+WSO2 products use asymmetric cryptography by default for authentication and data encryption. In asymmetric cryptography, keystores (with key pairs and certificates) are created and stored for the product. Keystore is a repository where private keys and certificates can be stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. For more information about keystores and its concepts, see [About Asymmetric Cryptography]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/about-asymetric-cryptography).
```
Comment (shaniR) [inline en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:3]: It is possible to have multiple keystores so that the keys used for different use cases are kept unique -> shouldn't this be "It is possible to have multiple keystores so that the keys used for different use cases are kept seperately"

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+In WSO2 API-M Analytics a set of keystores can be configured and used for different use cases in both dashboard and worker profiles.
+
+The `wso2carbon.jks` keystore file, which is shipped with all WSO2 products, is used as the default keystore for all functions. However, in a production environment, it is recommended to [create new keystores]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores) with new keys and certificates. If you have created a new keystore and updated the `client-truststore.jks` file, which resides in the `<API-M_ANALYTICS_HOME>/resources/security` directory, you must update the `deployment.yaml` file of the corresponding profile in order to make the keystore work.
+
+## Configuring keystores in dashboard profile
+
+!!! note
+    Other than explicitly mentioned, you need to do all the following configuration updates related to the dashboard profile in the `<API-M_ANALYTICS_HOME>/conf/dashboard/deployment.yaml` file.
+
+1. Update the Databridge configurations.
```
Comment (shaniR) [inline en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:14]: should Databrige be in camel case?

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+    This is used for authenticating communication over SSL/TLS between API Manager and API-M Analytics. Handles thrift and binary transports.
```
Comment (shaniR) [inline en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:16]: Authenticating the communication

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+      ... 
+      keyStoreLocation : ${sys:carbon.home}/resources/security/wso2carbon.jks
+      keyStorePassword : wso2carbon
+      ... 
+    ```
+    The elements in the above configuration are described below:
+       
+    | Element                | Description                                                      |
+    |------------------------|------------------------------------------------------------------|
+    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
```
Comment (shaniR) [inline en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:29]: should we mention about the formats supported as well?

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+          
+        | Element                | Description                                                      |
+        |------------------------|------------------------------------------------------------------|
+        | **privateKeyAlias**    | Private key alias                                                |
+        | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
+
+    2. Provide the keystore and private key password in the Base64 encoded format in the `<ANALYTICS_HOME>/conf/dashboard/master-keys.yaml` file.
+
+        ??? info "Click here for more info on the `master-keys.yaml` file"
+            The base64 encoded plaintext keystore password and the base64 encoded plaintext private password of the keystore used for Secure Vault is defined in the master-keys.yaml file. WSO2 Carbon Server reads this file to obtain these two passwords, so that it can decrypt the passwords in the `secrets.properties` file in order to start the server.
```
Comment (shaniR) [inline en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:79]: plain text -> two words?

Comment (shaniR) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:79]: Shall we reword this to, "The keystore password and private password which is in plain text is base64 encoded and stored in the master-keys.yaml"

Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:79]: Fixed.
Changed to 
"The keystore password and private password used for Secure Vault, which is in plain text is base64 encoded and stored in the `master-keys.yaml` file. "

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 4 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:29]: @fazlan-nazeem , please share your views.

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 2 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:14]: @fazlan-nazeem , please clarify.

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 1 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:3]: Fixed

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 3 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:16]: Fixed

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 2 above)
Comment (fazlan-nazeem) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:14]: DataBridge should be fine

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 4 above)
Comment (fazlan-nazeem) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:29]: do you mean other keystore file formats?

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 2 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:14]: Fixed

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 4 above)
Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:29]: @shaniR , please clarify @fazlan-nazeem 's Q

**Comment by Mariangela** [comment]
@shaniR , I have addressed all the comments that you have mentioned other that the comment where Fazlan had raised a Q on top of. Pls check and approve.

File: en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md (hunk 4 above)
Comment (bhathiya) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:29]: we support .jks only.

Comment (Mariangela) [reply en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md]: fixed

**Comment by Mariangela** [comment]
@shaniR , I updated this further. Please review and merge.
